app/Http/Controllers/EmployeeController.php

```diff
@@ -28,9 +28,11 @@ public function draw(Request $request){
         $draw = $request->query('draw', 1);
         $start = $request->query('start', 0);
         $length = $request->query('length', 10);
-        $totalEmployees =   Employee::count();
-        $arrayColumns = ['','company.name', 'company.branch', 'name', 'position', 'dob', 'email', 'phone', 'address', 'bankAccount.account_no'];
+        $totalEmployees = Employee::count();
+        $arrayColumns = ['', 'company_name', 'branch', 'name', 'position', 'dob', 'email', 'phone', 'address', 'account_no'];
```

Comment:
remove this hard coded array. think each time when we add a new column to the table. we have to assign here

Reply:
https://github.com/aanish27/employee-profile-manager/pull/83#discussion_r1819856828